catalogs/make_1LHAASO_fits.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jun 15 17:15:06 2023
+
+@author: qremy
+"""
```

Comment:
Add the used version of Gammapy, that fixes the versions of numpy, astropy and pandas

Reply:
added to header via the CREATOR keyword